documents/role-description-developer.md

```diff
@@ -0,0 +1,29 @@
+# Rollebeskrivelse for programvareutvikler
+
+Målet med beskrivelsen er å trekke opp noen rammebetingelser for din rolle i
+Bjerk. Samtidig som den enkeltes hovedarbeidsområde beskrives, legges det hos
+Bjerk stor vekt på samarbeid og at alle bidrar gjennom sin definerte rolle i
+selskapet.
+
+## Ansvarsområder og rolle
+
+Som programvareutvikler hos Bjerk er følgende dine ansvarsområder:
+
+- Utvikle programvare
+- Bidra til utvikling av programvare
```

Comment:
Litt vanskelig å forstå forskjellen mellom disse to linjene kanskje?

Reply:
Hva syntes du?